payments/tests/test_reservation_api.py

```diff
@@ -495,124 +495,143 @@ def test_order_must_include_rent_if_one_exists(user_api_client, paid_resource):
+    "params",
     (
-        # NORMAL, no authorization, has order, OK+payment
-        (Reservation.TYPE_NORMAL, None, True, True, Reservation.WAITING_FOR_PAYMENT, 1),
-        # INTERNAL USE, no authorization, has order, FAIL
-        (Reservation.TYPE_INTERNAL_USE, None, True, False, None, 0),
-        # NORMAL, no authorization, no order, FAIL
-        (Reservation.TYPE_NORMAL, None, False, False, None, 0),
-        # INTERNAL USE, no authorization, no order, FAIL
-        (Reservation.TYPE_INTERNAL_USE, None, False, False, None, 0),
-        # NORMAL, viewer, has order, OK+payment
-        (
-            Reservation.TYPE_NORMAL,
-            UnitAuthorizationLevel.viewer,
-            True,
-            True,
-            Reservation.WAITING_FOR_PAYMENT,
-            1,
-        ),
-        # INTERNAL USE, viewer, has order, FAIL
-        (
-            Reservation.TYPE_INTERNAL_USE,
-            UnitAuthorizationLevel.viewer,
-            True,
-            False,
-            None,
-            0,
-        ),
-        # NORMAL, viewer, no order, FAIL
-        (Reservation.TYPE_NORMAL, UnitAuthorizationLevel.viewer, False, False, None, 0),
-        # INTERNAL USE, viewer, no order, FAIL
-        (
-            Reservation.TYPE_INTERNAL_USE,
-            UnitAuthorizationLevel.viewer,
-            False,
-            False,
-            None,
-            0,
-        ),
-        # NORMAL, manager, has order, OK+payment
-        (
-            Reservation.TYPE_NORMAL,
-            UnitAuthorizationLevel.manager,
-            True,
-            True,
-            Reservation.WAITING_FOR_PAYMENT,
-            1,
-        ),
-        # NORMAL, manager, no order, FAIL
-        (
-            Reservation.TYPE_NORMAL,
-            UnitAuthorizationLevel.manager,
-            False,
-            False,
-            None,
-            0,
-        ),
-        # INTERNAL USE, manager, no order, OK+confirmed
-        (
-            Reservation.TYPE_INTERNAL_USE,
-            UnitAuthorizationLevel.manager,
-            False,
-            True,
-            Reservation.CONFIRMED,
-            0,
-        ),
-        # NORMAL, admin, has order, OK+payment
-        (
-            Reservation.TYPE_NORMAL,
-            UnitAuthorizationLevel.admin,
-            True,
-            True,
-            Reservation.WAITING_FOR_PAYMENT,
-            1,
-        ),
-        # NORMAL, admin, no order, FAIL
-        (
-            Reservation.TYPE_NORMAL,
-            UnitAuthorizationLevel.admin,
-            False,
-            False,
-            None,
-            0,
-        ),
-        # INTERNAL USE, admin, no order, OK+confirmed
-        (
-            Reservation.TYPE_INTERNAL_USE,
-            UnitAuthorizationLevel.admin,
-            False,
-            True,
-            Reservation.CONFIRMED,
-            0,
-        ),
+        {
+            "reservation_type": Reservation.TYPE_NORMAL,
```

Comment:
passing a `dict` rather than tuple of params to `parametrize` makes code a bit easier to read without need for comments.